src/utils/tracking.tsx

```diff
@@ -22,4 +22,12 @@ export const trackDrawerStatus = (action: boolean): void => {
   });
 };
 
+export const trackUserLocation = (lat: number, lon: number): void => {
+  ReactGA.event({
+    category: 'Geolocation',
+    action: 'Set',
+    label: `${lat},${lon}`,
```

Comment:
Just want to make sure this actually makes sense in GA

Reply:
LGTM, though I suspect we'll need to see some of the data come through before we know for sure.